<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequestInputWidget</class>
 <widget class="QWidget" name="RequestInputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>156</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>197</width>
    <height>101</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="awayLabel">
          <property name="text">
           <string>Away</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="awayScoreInput"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="homeLabel">
          <property name="text">
           <string>Home</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="homeScoreInput"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="timeLabel">
          <property name="text">
           <string>Time</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="timeInput"/>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="periodLabel">
          <property name="text">
           <string>Period</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="periodInput"/>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QPushButton" name="btnRequestStats">
          <property name="text">
           <string>Request Stats</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
